package.json

```diff
@@ -25,17 +27,25 @@
     "url": "https://github.com/Pomax/react-onclickoutside/issues"
   },
   "scripts": {
+    "build:es": "rimraf es && cross-env BABEL_ENV=es babel src --out-dir es",
+    "build:cjs": "rimraf lib && cross-env BABEL_ENV=cjs babel src --out-dir lib",
+    "build:umd:dev": "cross-env BABEL_ENV=es NODE_ENV=development rollup -c -i src/index.js -o dist/react-onclickoutside.js",
+    "build:umd:prod": "cross-env BABEL_ENV=es NODE_ENV=production rollup -c -i src/index.js -o dist/react-onclickoutside.min.js",
+    "build:umd": "rimraf dist && npm run build:umd:dev && npm run build:umd:prod",
+    "build": "npm run build:es && npm run build:cjs && npm run build:umd",
```

Comment:
let's not generate all these separate files. Just rollup the source to a umd bundle and we're done.